src/core/atomic.d

```diff
@@ -466,7 +466,7 @@ in (atomicPtrIsProperlyAligned(here), "Argument `here` is not properly aligned")
 *  true if the store occurred, false if not.
 */
 bool casWeak(MemoryOrder succ = MemoryOrder.seq,MemoryOrder fail = MemoryOrder.seq,T,V)(T* here, T* ifThis, V writeThis) pure nothrow @nogc @trusted
-if (!is(T == shared S, S) && !is(V == shared U, U))
+    if (!is(T == shared S, S) && !is(V == shared U, U))
```

Comment:
Before your change this was the official DStyle (dlang.org/dstyle).

Reply:
It was inconsistent with the rest of the file... I would always favour internal consistency :/

Reply:
We have a style guide for a reason!!